keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -291,113 +697,285 @@ No, the default behavior remains the same.
+2. Upgrade to 1.34.
+3. Create StatefulSet A with spec.updateStrategy.rollingUpdate.maxUnavailable set to 3, with 6 replicas
+4. Verify a rollout and check if only 3 pods are unavailable at a time
+5. Check if rollout is also fine with podManagementPolicy set to Parallel
+6. Downgrade to 1.33.
+7. Verify that the rollout only has 1 pod unavailable at a time, similar to setting maxUnavailable to 1 (MaxUnavailableStatefulSet feature gate disabled by default).
+8. Create another StatefulSet B not setting maxUnavailable (leaving it nil)
+9. Upgrade to 1.34.
+10. Verify that the rollout has default behavior of only having one pod unavailable at a time
+    Verify that the `maxUnavailable` can be set again to StatefulSet A and test the rollout behavior
 
 ###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
 No
 
 ### Monitoring Requirements
 
 ###### How can an operator determine if the feature is in use by workloads?
+
 If their StatefulSet rollingUpdate section has the field maxUnavailable specified with
-a value different than 1.
-The below command should show maxUnavailable value:
+a value different than 1. While in alpha and beta, the feature-gate needs to be enabled.
+
+The command bellow should show the maxUnavailable value:
+
 ```
 kubectl get statefulsets -o yaml | grep maxUnavailable
 ```
 
 ###### How can someone using this feature know that it is working for their instance?
-TODO when we reach Beta
+
+Users can verify the maxUnavailable feature is working for their StatefulSets by observing the behavior of their application during rolling updates. Specifically, they should monitor the update progress of StatefulSets against the maxUnavailable value they have configured. This can be done by tracking the number of pods that are unavailable at any given time during the update process and ensuring it does not exceed the specified maxUnavailable limit. Users can also view the `statefulset_unavailability_violation` metric to see if there have been instances
+where the feature is not working as intended.
 
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+Startup latency of schedulable stateful pods should follow the [existing latency SLOs](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/slos.md#steady-state-slisslos).
+
```

Comment:
How about stating a reasonable SLO for that new `statefulset_unavailability_violation` metric?